src/views/Dashboard.vue

```diff
@@ -293,6 +327,9 @@ export default {
       if (this.isBComp()) {
         this.checkPendingFilings(this.historyFilings)
       }
+      // check for completed filing on an incoporation application
+      this.checkCompletedIncorporationFilings(this.historyFilings)
```

Comment:
I suggest adding `if (this.nrNumber)` to this which will do less work (if not NR) and remove the need for the NR check on line 183 (and probably remove the need for the computed property "completedFilingRequired").

Reply:
added the NR check, but after taking a look. I feel like I can do a bit more clean up based on your feed back about not needing the computed property.

Will commit another update in a bit.

Reply:
Updated.
- removed computed completedFilingRequired
- renamed variable hasCompletedIncorporationFIling to completedFilingRequired as we are more interested in the fact that is is required for the list components